pkgs/misc/emulators/yuzu/base.nix

```
@@ -0,0 +1,66 @@
+{ pname, version, src, branchName
+, stdenv, fetchFromGitHub, wrapQtAppsHook
+, cmake, pkgconfig
+, libpulseaudio, libjack2, alsaLib, sndio, ecasound
+, vulkan-loader, vulkan-headers
+, qtbase, qtwebengine, qttools
+, nlohmann_json, rapidjson
+, zlib, zstd, libzip, lz4
+, glslang
+, boost173
+, catch2
+, fmt
+, SDL2
+, udev
+, libusb1
+, ffmpeg
+}:
+
+stdenv.mkDerivation rec {
+  inherit pname version src;
+
+  nativeBuildInputs = [ cmake pkgconfig wrapQtAppsHook ];
```

```suggestion
  nativeBuildInputs = [ cmake pkg-config wrapQtAppsHook ];
```

pkgconfig is deprecated and now causes an eval error.